doc/porting_guide.md

```diff
@@ -444,18 +458,18 @@ While we cannot completely predict how this process would work for every IDE and
 - import all of the source files in one of the subdirectories of `src/bsp/platform`.
   The Xively C Client BSPs contain all of the missing hooks which tie the platform independent
   code to a particular device. Complete implementations should include a source file for each
-  of the BSP subsystems (networking, memory, time, etc).
-	- NOTE: The Xively C Client contains reference BSP implementations for POSIX and CC3200. We also have partial implementations for specific networking APIs.
+  of the BSP subsystems (sft, networking, memory, time, etc).
```

Comment:
there is not SFT BSP yet. "just" IO_FS and FWU. Although maybe the FWU should be separated into two parts: SFT and FWU but for simplicity reasons it's good to have one even if it isn't 100% correct.